app/views/spree/admin/users/_form.html.haml

```diff
@@ -14,7 +14,7 @@
       = hidden_field_tag "user[spree_role_ids][]", ""
     = f.field_container :locale do
       = f.label :locale, t(".locale")
-      = f.select :locale, locale_options, class: "fullwidth"
+      = f.select :locale, OpenFoodNetwork::I18nConfig.locale_options, class: "fullwidth"
```

Comment:
I think we are better off fixing the issue here : 
https://github.com/openfoodfoundation/openfoodnetwork/blob/0c43dd4222ecdf6403f38e710c286f39364b5b88/lib/open_food_network/i18n_config.rb#L19

The helper function itself isn't wrong ( https://github.com/openfoodfoundation/openfoodnetwork/blob/0c43dd4222ecdf6403f38e710c286f39364b5b88/app/helpers/i18n_helper.rb#L4 )

Reply:
Hi @rioug 
I am curious as to what the fix here should be.
I simply used  local_options as per this [comment](https://github.com/openfoodfoundation/openfoodnetwork/issues/11513#issuecomment-2092359716)

Initially, this issue was about removing duplicate English from the dropdown menu list, where @sergioosouzaa removed the `souce_local` that is appended to the list of the available locales (later reverted [here](https://github.com/openfoodfoundation/openfoodnetwork/pull/12434/files)).

Do I understand correctly that we want to show just one English (which should be thesame for other languages with multiple locals eg `fr`, `es`) in the dropdown irrespective of the number of `en` related locals configured?
eg With this configuration `AVAILABLE_LOCALES="en_AU,es,en_US,en_BE,en_CA,es_CO,es_CR,es_US,fr_CH,fr_CA,fr_CM,fr_BE"`, I would expect to see just 3 languages (English, Francais, and Espanol) in the selector?

Reply:
Fair enough I missed the comment. 
I did not quite grasp the problem. From looking at it again, it looks to me to be a translation problem. For instance all the English locales are named "English" that's why we get a duplicates, so I reckon we should be fixing the translations instead. 

Before moving forward, let's clarify what they meant,@mkllnk @dacook by [this](https://github.com/openfoodfoundation/openfoodnetwork/issues/11513#issuecomment-2092359716) did you mean using non translated locale in the dropdown ? ie having something like en_AU, en_CA, fr etc.. as options ? or did you mean using  `locale_options` in the helper here :
https://github.com/openfoodfoundation/openfoodnetwork/blob/0c43dd4222ecdf6403f38e710c286f39364b5b88/app/helpers/i18n_helper.rb#L4-L8

Reply:
I think the issue description was a bit misleading, and the method names are even more confusing.
This is what I meant: https://github.com/dacook/openfoodnetwork/commit/4a9a42e7b922430fbc599f0ba911a91f37ed1aa6

Wandji could you please try that and test if it solves the problem?

Reply:
@dacook 
Tested the diff and now we do not get the `en` locale in the list
However, we still have multiple languages displayed with thesame label

I agree with @rioug that we should be fixing the translation and still use the `selectable_locales` here

![image](https://github.com/user-attachments/assets/371c4c73-5439-4869-a710-42b430f3f140)

Reply:
Regarding the multiple "English", (or Espanol, etc), this is not a problem. It is correctly showing en, en_AU, en_UK etc. Only one of these is ever enabled on a system (if we need to enable more, then we can change the label).

The problem we're trying to solve is simply that the `en` language always appears, even when it is not enabled.

Reply:
> I agree with rioug that we should be fixing the translation and still use the `selectable_locales` here

I don't understand, do you mean that we should update `i18n_config.rb` (instead of `i18n_helper.rb` as I suggested)? As as far as I understand, `available_locales` has a different purpose and shouldn't be changed (as per the commnt): 
https://github.com/openfoodfoundation/openfoodnetwork/blob/0c43dd4222ecdf6403f38e710c286f39364b5b88/lib/open_food_network/i18n_config.rb#L13-L21

Reply:
> The problem we're trying to solve is simply that the en language always appears, even when it is not enabled.

For this, I have updated as discussed to use selectable_options which resolves the issue.


As far as fixing translations, I would suggest updating `language_name` to match the full language name where possible e.g. In `config/locales/en_AU.yml` we could change language_name: "English" to language_name: "English - AUSTRALIA (AU)"

Reply:
Thanks.
So far we've never needed to have multiple English locales used at the same time (eg in Australia we only enable en_AU; in UK it's only en_UK), so we don't need to update the language_name at this time.